src/components/search_bar/search_bar.tsx

```diff
@@ -99,6 +99,12 @@ export interface OuiSearchBarProps extends CommonProps {
   dateFormat?: object;
 
   compressed?: boolean;
+
+  // @deprecated This property has never been consumed despite being defined
+  'aria-label'?: string;
+
+  // @deprecated This property has never been consumed despite being defined
+  'data-test-subj'?: string;
```

Comment:
do we still have a `data-test-subj` that we can use? for our tests?

Reply:
the actual component inside does and that one has its own test id. These two have never been attached to anything.